lib/board.rb

```diff
@@ -3,15 +3,14 @@
 class Board
   attr_reader :cells, :letter_rows, :num_columns
 
-  def initialize(letter_rows = 4, num_columns = 4)
-    @letter_rows = letter_rows
-    @num_columns = num_columns
+  def initialize(board_size = 4)
+    @letter_rows = board_size
+    @num_columns = board_size
```

Comment:
I'm wondering if we can just replace @letter_rows and @num_columns with @board_size completely and change each instance in the class to @board_size as letter_rows and num_columns now are a bit obsolete. Let me know what you think!

Reply:
Love that idea. I'll add that to the ideas to address tonight or tomorrow!